scaleway/provider.go

```diff
@@ -149,6 +149,7 @@ func Provider(config *ProviderConfig) plugin.ProviderFunc {
 				"scaleway_lb_route":                            resourceScalewayLbRoute(),
 				"scaleway_registry_namespace":                  resourceScalewayRegistryNamespace(),
 				"scaleway_tem_domain":                          resourceScalewayTemDomain(),
```

Comment:
Could this logic be implemented directly inside the `scaleway_tem_domain` ressource?

Reply:
No, because to validate the domain, we have to make records that depend on the scaleway_tem_domain resource. See => https://registry.terraform.io/providers/scaleway/scaleway/latest/docs/resources/tem_domain#add-the-required-records-to-your-dns-zone. We must wait for the records on the domain before the domain_tem verification.